Sources/Clappr/Classes/Base/Layers/LayerComposer.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+
+public class LayerComposer {
+    private weak var rootView: UIView?
+    private let backgroundLayer = BackgroundLayer()
+    private let playbackLayer = PlaybackLayer()
+    private let containerLayer = ContainerLayer()
+    
+    public init(){}
```

Comment:
Maybe we can omit this init.

```suggestion
```

Reply:
As LayerComposer is a parameter to Core and Container required initializers, and those initializers are public, we need to declare it as a public class also. And as such, we need to enable the consumers of our lib to instantiate it, or else they won't be able to use Core or Container's initializer also.

Reply:
Got it.
;)